canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -68,12 +74,15 @@ $toolbar-divider-width: 1px;
 	&.ghost {
 		& button {
 			background-color: $ui-01;
+			padding: 0;
+			.toolbar-icon-label.before {
```

Comment:
The styling here does not look correct, the label and icon should be vertically centered. We need to also use the Carbon space tokens instead of specifying an arbitrary value like 5 or 6

<img width="877" alt="image" src="https://user-images.githubusercontent.com/1628100/191578523-5871d0ab-16e9-464d-86df-235c1506e7c6.png">